packages/client/components/Dashboard/DashSidebar.tsx

```diff
@@ -154,6 +154,7 @@ const DashSidebar = (props: Props) => {
           <DashHR />
           <NavMain>
             <NavList viewer={viewer} />
+            <NavList viewer={viewer} />
```

Comment:
how does it work? We now have 2 same components?

Reply:
I added that to reproduce the UI issue you mentioned. Thanks, removed!